cmd/check_operator.go

```diff
@@ -31,19 +33,19 @@ var checkOperatorCmd = &cobra.Command{
 		}
 
 		if len(args) != 1 {
-			return fmt.Errorf("An operator image positional argument is required")
+			return fmt.Errorf("an operator image positional argument is required")
 		}
 		return nil
 	},
 	// this fmt.Sprintf is in place to keep spacing consistent with cobras two spaces that's used in: Usage, Flags, etc
 	Example: fmt.Sprintf("  %s", "preflight check operator quay.io/repo-name/operator-bundle:version"),
+	PreRun:  preRunConfig,
```

Comment:
This shouldn't be here.

Reply:
Fixed. It wasn't flagged as removed in my diff, and was just hanging out there in the wrong place.